vendor/bat-native-ads/src/bat/ads/internal/static_values.h

```diff
@@ -18,7 +18,8 @@ namespace ads {
 #define STAGING_SERVER "https://ads-serve.bravesoftware.com"
 #define DEVELOPMENT_SERVER "https://ads-serve.brave.software"
 
-#define CATALOG_PATH "/v2/catalog"
+const int kCatalogVersion = 2;
+#define CATALOG_PATH "/v"kCatalogVersion"/catalog"
```

Comment:
Honestly, I never seen such a construction. Did you check, does it replace it correctly? As to me it would replace it like
`#define CATALOG_PATH "/v"2"/catalog"`

Reply:
yeap fixed now, as I did not push correct commits